packages/message/package.json

```diff
@@ -25,8 +25,8 @@
     "react": "^16.12.0"
   },
   "dependencies": {
-    "@emotion/core": "^10.0.27",
-    "@emotion/serialize": "^0.11.15",
+    "@emotion/core": "^10.0.28",
```

Comment:
We just need to check this version of emotion carefully on the jilt side but maybe theme-ui 0.3.1 fixes the other problems we saw from upgrading?